packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -163,12 +163,7 @@ export enum ChainId {
   StylusTestnet = 23011913
 }
 
-// allow only Ethereum testnets and Arbitrum testnets as parent chains
-export const allowedParentChainIds = [
-  ChainId.ArbitrumGoerli,
-  ChainId.ArbitrumNova,
-  ChainId.ArbitrumLocal
-]
+export const supportedCustomOrbitParentChains = [ChainId.ArbitrumGoerli]
```

Comment:
Let's add Arbitrum Sepolia here's since we're enabling that very soon. 
Is there a reason we don't include Arbitrum One and Nova? I know we don't have mainnet Orbit chains right now, but they will exist at some point.